test/610_proxy_wait_for_weave_test.sh

```diff
@@ -19,4 +19,7 @@ COMMITTED_IMAGE=$(proxy docker_on $HOST1 commit c1)
 assert_raises "proxy docker_on $HOST1 run --name c2 $COMMITTED_IMAGE"
 assert "entrypoint c2" "$(entrypoint $COMMITTED_IMAGE)"
 
+# Check containers get their weave IP prepended to /etc/hosts
```

Comment:
Maybe better to comment the effect rather than the mechanism (which isn't apparent here nor important)